enterprise/server/execution_service/execution_service.go

```diff
@@ -32,9 +35,16 @@ func checkPreconditions(req *espb.GetExecutionRequest) error {
 	return status.FailedPreconditionError("An execution lookup with invocation_id must be provided")
 }
 
-func (es *ExecutionService) queryExecutions(ctx context.Context, baseQuery *query_builder.Query) ([]*tables.Execution, error) {
+func (es *ExecutionService) getInvocationExecutions(ctx context.Context, invocationID string) ([]*tables.Execution, error) {
```

Comment:
The diff here looks a little weird, but it's getting rid of `queryExecutions` and moving that logic into `getInvocationExecutions` since it wasn't called anywhere else, and changing the invocations JOIN to be a LEFT JOIN instead.